### DISCUSSION TITLE
docker compose example  - clickhouse- kafka-connect

### DISCUSSION THREAD

**Discussion by robinjohn29** [ClickHouse/clickhouse-kafka-connect#152] [Q&A] [answered]
Is there an example of adding clickhouse-sink-connector plugin to kafka connect via docker-compose that can be shared?

**Comment by Paultagoras** [comment, marked as answer]
Hi @robinjohn29 - the bits you need are:

`volumes:
      - "~/DockerShare:/usr/share/dockershare/:ro"`

and

`CONNECT_PLUGIN_PATH: "/usr/share/java,/usr/share/confluent-hub-components,/usr/share/dockershare"`

That maps a folder (in the example, called DockerShare on my local mac) and adds it to the plugin path.
